server/command/command.go

```diff
@@ -746,3 +777,54 @@ func (c *Handler) executeTestCommand(_ *model.CommandArgs) *model.CommandRespons
 		Text:         responseText.String(),
 	}
 }
+
+func (c *Handler) executeMigrateCommand(_ *model.CommandArgs) *model.CommandResponse {
+	// Get current version before reset
+	kvstore := c.plugin.GetKVStore()
+	versionBytes, _ := kvstore.Get("kv_store_version")
+	currentVersion := "0"
+	if len(versionBytes) > 0 {
+		currentVersion = string(versionBytes)
+	}
+
+	// Reset KV store version to 0 to force re-migration
+	if err := kvstore.Set("kv_store_version", []byte("0")); err != nil {
+		return &model.CommandResponse{
+			ResponseType: model.CommandResponseTypeEphemeral,
+			Text:         fmt.Sprintf("❌ Failed to reset migration version: %v", err),
+		}
+	}
+
+	// Run migrations and get detailed results
+	migrationResult, err := c.plugin.RunKVStoreMigrationsWithResults()
+	if err != nil {
+		return &model.CommandResponse{
+			ResponseType: model.CommandResponseTypeEphemeral,
+			Text:         fmt.Sprintf("❌ Migration failed: %v", err),
+		}
+	}
+
+	// Get the results from migration
+	userMappingsAdded := migrationResult.UserMappingsCreated
+	channelMappingsAdded := migrationResult.ChannelMappingsCreated
+	roomMappingsAdded := migrationResult.RoomMappingsCreated
+	dmMappingsAdded := migrationResult.DMMappingsCreated
+	reverseDMMappingsAdded := migrationResult.ReverseDMMappingsCreated
+
+	return &model.CommandResponse{
+		ResponseType: model.CommandResponseTypeEphemeral,
+		Text: fmt.Sprintf("✅ **Migration completed successfully!**\n\n"+
+			"**Migration Results:**\n"+
+			"   • Reset version: %s → 2\n"+
```

Comment:
Last version from variable?

Reply:
Ah, yes.  I'll fix that.

Reply:
Fixed via https://github.com/mattermost/mattermost-plugin-matrix-bridge/pull/8